libminifi/include/utils/Cron.h

```diff
@@ -36,6 +36,7 @@ class CronField {
   virtual ~CronField() = default;
 
   [[nodiscard]] virtual bool matches(date::local_seconds time_point) const = 0;
+  virtual bool operator==(const CronField&) const { throw std::runtime_error("not implemented"); }
```

Comment:
Why not pure virtual?

Reply:
There are a bunch more derived classes and I didn't want to implement it for all, since the comparison only makes sense for tests.